### PR TITLE
Advancing 0.13.0-rc2 release to status: installers

### PR DESCRIPTION
**Pull request by dfarrell07**
Advancing 0.13.0-rc2 release to status: installers

### CODE REVIEW

File: releases/v0.13.0-rc2.yaml [hunk 1]
```diff
@@ -3,10 +3,13 @@ version: v0.13.0-rc2
 name: 0.13.0-rc2
 pre-release: true
 branch: release-0.13
-status: projects
+status: released
```
Comment (tpantelis) [inline releases/v0.13.0-rc2.yaml]: Shouldn't the next phase be [installers](https://github.com/submariner-io/releases/pull/422/files)?

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: releases/v0.13.0-rc2.yaml (hunk 1 above)
Comment (dfarrell07) [reply releases/v0.13.0-rc2.yaml]: That would makes sense, yes.

This is what `make release VERSION="0.13.0-rc2"` gave me, but let me try again.

Comment (dfarrell07) [reply releases/v0.13.0-rc2.yaml]: I ran it again and it updated this PR with what seems to be the correct phase.

I guess it was because I ran it the first time with a GH token with the wrong permissions, and then re-ran with a correct token.

File: releases/v0.13.0-rc2.yaml [hunk 2]
```diff
@@ -3,10 +3,12 @@ version: v0.13.0-rc2
...
 branch: release-0.13
-status: projects
+status: installers
 components:
   shipyard: 9b91c1e69cea262c5da51d20f1749d3ecb485246
   admiral: 2239a501d8c1a03be1b7b08f722e74a8d32d2ca8
   cloud-prepare: a4dda976d643d2cb5283e4b7cacf06341cf6d130
   lighthouse: c454961ea91af3bc45a1f40e157083f32269049c
   submariner: 7531271eaffb81219917dcb14a9fcebc3f498b68
+  submariner-charts: dc3f31ab888b172f31d0a53c9572e67e80e648df
```
Comment (dfarrell07) [inline releases/v0.13.0-rc2.yaml:13]: https://github.com/submariner-io/submariner-charts/commit/dc3f31ab888b172f31d0a53c9572e67e80e648df

File: releases/v0.13.0-rc2.yaml [hunk 3]
```diff
@@ -3,10 +3,12 @@ version: v0.13.0-rc2
...
+  submariner-operator: daa3a5268a491805d40f8e1bdac8fd50d43999e7
```
Comment (dfarrell07) [inline releases/v0.13.0-rc2.yaml:14]: https://github.com/submariner-io/submariner-operator/commit/daa3a5268a491805d40f8e1bdac8fd50d43999e7

**Review by dfarrell07 — COMMENTED** [review]
SHAs look good

**Review by tpantelis — APPROVED** [review]

**Review by aswinsuryan — APPROVED** [review]

**Review by maayanf24 — APPROVED** [review]
